### PR TITLE
Fmu pwm limit

### PR DESCRIPTION
**Pull request by kd0aij**
fmu was asserting armed input to pwm_limit_calc prematurely

### CODE REVIEW

File: src/drivers/px4fmu/fmu.cpp [hunk 1]
```diff
@@ -789,14 +789,10 @@ PX4FMU::cycle()
 		orb_copy(ORB_ID(actuator_armed), _armed_sub, &_armed);
 
 		/* update the armed status and check that we're not locked down */
-		bool set_armed = (_armed.armed || _armed.prearmed) && !_armed.lockdown;
```
Comment (LorenzMeier) [inline src/drivers/px4fmu/fmu.cpp]: The pre-armed state is needed for applications which allow to move e.g. the ailerons once the button is pressed. So most likely the issue here is _armed.prearmed set in the wrong state? Is it maybe always true?

**Comment by kd0aij** [comment]
bench tested

**Comment by LorenzMeier** [comment]
Cool. Could you bench test that with your VTOL and safety switch as well? That is the corner case we need to make sure to not break. If you push the safety switch you should get attitude control on the ailerons, but no throttle.

**Comment by kd0aij** [comment]
pre-arming works as before on fmuV2 VTOL; servos active on pressing safety switch
Throttle activates the same as before, motors spin briefly on arming via RC, stop then start again near end of arming tone. 
After disarming, the motors spin very briefly or not at all before the end of the arming tone.
But I believe the code responsible for for fmuV2 behavior is in the IO coprocessor, not drivers/px4fmu code.
